lib/data/repository/movies_repository_impl.dart

```diff
@@ -35,9 +40,14 @@ class MoviesRepository with DatabaseResponse implements IMoviesRepository {
         var apiResponse = await _service.apiCall(endpoint: endpoint);
         if (apiResponse.statusCode == HttpStatus.ok) {
           List<dynamic> movies = json.decode(apiResponse.body)['results'];
+
           if (movies.isEmpty) {
             return MovieEvent(status: Status.empty);
           } else {
+            movies.forEach((movie) {
+              (movie as Map<String, dynamic>)['lowerCaseTitle'] =
+                  (movie['title'] as String).toLowerCase();
+            });
             _db.addMovie(
               movies: movies,
               mainCollectionDocument: document,
```

Comment:
![image](https://user-images.githubusercontent.com/73655171/197789255-bb682f57-ed24-42af-a672-384407eb2265.png)

Reply:
I think you can rename db method because you are add more than one movie

Reply:
Agreed, it takes a list instead of a single movie. Renamed to addMovies.